source/UIAHandler/__init__.py

```diff
@@ -68,15 +68,14 @@
 # also exposes this value as the document UIA element's classname property.
 MS_WORD_DOCUMENT_WINDOW_CLASS = "_WwG"
 
-HorizontalTextAlignment_Left=0
-HorizontalTextAlignment_Centered=1
-HorizontalTextAlignment_Right=2
-HorizontalTextAlignment_Justified=3
-
+HorizontalTextAlignment_Left = 0
+HorizontalTextAlignment_Centered = 1
+HorizontalTextAlignment_Right = 2
+HorizontalTextAlignment_Justified = 3

```

Comment:
If you are touching this part of code, please make it in an enum

Reply:
This will break NVDAObjects.UIA text alignment code, so I would be careful with this one.

Reply:
I can add a "getattr" function at the end in case we do find add-ons using UIAHandler.Horizontal* attributes (and perhaps a good test case for structured pattern matching). Will add an entry to changelog file.